source/components/inputs/select/select.ng1.ts

```diff
@@ -96,10 +96,8 @@ export class SelectController extends InputController {
 		this.showOptions = !this.showOptions;
 	}
 
-	close: { (): void } = () => {
-		if (this.showOptions) {
```

Comment:
It was lost while trying different things and the newer method  worked how I wanted it. Is there an advantage to the original method versus mine?